rollup.config.js

```diff
@@ -1,12 +1,15 @@
 // rollup.config.js
 import dts from 'rollup-plugin-dts';
+import sourcemaps from 'rollup-plugin-sourcemaps';
 const config = [
   {
     input: './dist/tmp/index.js',
     output: {
       file: 'dist/index.js',
       format: 'es',
+      sourcemap: true,
     },
+    plugins: [sourcemaps()],
```

Comment:
You're going to want to mark `react-native` as an external dependency since its a peerDependency (and it'll also remove the rollup warning).

```suggestion
    input: './dist/tmp/index.js',
    output: {
      file: 'dist/index.js',
      format: 'es',
      sourcemap: true,
    },
    external: [ 'react-native' ],
    plugins: [sourcemaps()],
```